src/_pytest/reports.py

```diff
@@ -196,11 +196,17 @@ def head_line(self) -> Optional[str]:
             return domain
         return None
 
-    def _get_verbose_word(self, config: Config):
+    def _get_verbose_word(
+        self, config: Config, default_markup: Mapping[str, bool]
+    ) -> Tuple[str, Mapping[str, bool]]:
```

Comment:
The pre-commit run here https://results.pre-commit.ci/run/github/37489525/1719242655.jVEQ62HET5qmvF-sLT8qtA exposes that `Tuple` hasn't been imported. Though, perhaps a lowercase variant would work, since Ronny pushed some typing improvements last week:
```suggestion
    ) -> tuple[str, Mapping[str, bool]]:
```

Reply:
@pbrezina it used to be imported last week. But Ronny contributed a huge refactoring with future imports that enabled the project to use newer syntax. And dropping the imports was a part of that, as they became unnecessary.

You created your branch just before that happened, so you don't see the changes in your fork. But if you rebase on top of the upstream main branch, you'll see that the imports are no longer there.